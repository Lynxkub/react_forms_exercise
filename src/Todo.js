import React from 'react';


const Todo = ({todo , id , deleteTodo}) =>  {

    const handleClick = () => {
        deleteTodo(id);
    }
    return (
        <div>
            <div style = {{display : 'inline-block'}}>{todo}</div>
            <button onClick = {handleClick}>Remove Todo</button>
        </div>
    )
}

export default Todo;