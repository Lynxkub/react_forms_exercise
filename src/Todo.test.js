import React from 'react';
import {render , fireEvent} from '@testing-library/react'
import Todo from './Todo';
import '@testing-library/jest-dom/extend-expect';


it('renders without crashing' , function () {
    render(<Todo />)
})


it('matches snapshot' , function () {
    const {asFragment} = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
})