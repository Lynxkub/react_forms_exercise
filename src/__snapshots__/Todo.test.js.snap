// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      style="display: inline-block;"
    />
    <button>
      Remove Todo
    </button>
  </div>
</DocumentFragment>
`;
